src/components/AddMovie.jsx

```diff
@@ -1 +1,101 @@
-// implement AddMovie component here
+import React from 'react';
+import PropTypes from 'prop-types';
+import InputGeneric from './InputGeneric';
+import TextArea from './TextArea';
+import SelectElement from './SelectElement';
+import Button from './Button';
+
+export default class AddMovie extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      subtitle: '',
+      title: '',
+      imagePath: '',
+      storyline: '',
+      rating: 0,
+      genre: 'action',
+    };
+  }
+
+  hanbleOnSubmit = () => {
```

Comment:
```suggestion
  hanbleOnSubmit = (event) => {
    event.preventDefault();
    const { onClick } = this.props;
    onClick(this.state);
    this.setState({
      subtitle: '',
      title: '',
      imagePath: '',
      storyline: '',
      rating: 0,
      genre: 'action',
    });
  };
```
acho que não é preciso usar uma captura pelo dom dessa forma no react, ao meu ver da para arrumar outras maneira com o hook eo state do react